src/components/pages/ChooseTicketsPage.vue

```diff
@@ -1,10 +1,8 @@
-<script setup>
-const props = defineProps({
-  seatsSelected: {
-    type: Array,
-    required: true
-  }
-})
+<script lang="ts" setup>
+
+const props = defineProps<{
+  seatsSelected: Array<string>
```

Comment:
You can also use the shorthand if you like it more 😄
The same goes for all the other places.

```suggestion
  seatsSelected: string[]
```

Reply:
I actually have tried both syntaxes on purpose -- as an exercise 😉 I know it looks messy to have both in the code -- I will change it for the shorthand everywhere.

Reply:
Oh, it makes sense then 😄 Great!